src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRule.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents a Mapping Rule in the Topic Mapping Rules(TOMAR). Mapping rules are used to define how MQTT topics should be mapped to Kafka topics.
+ * E.g.: a valid mapping rule would look like this in the TOMAR file:
+ * {
+ * "kafkaTopic": "sensors_{sensorId}_data",
+ * "mqttTopic": "sensors/{sensorId}/data"
+ * }
+ * and like this in the MappingRule class:
+ * MappingRule(kafkaTopicTemplate=sensors_{sensorId}_data mqttTopicPattern= sensors/{sensorId}/data)
+ */
+public class MappingRule {
+    @JsonProperty("kafkaTopic")
+    private String kafkaTopicTemplate;
+    @JsonProperty("mqttTopic")
+    private String mqttTopicPattern;
+
+    /**
+     * Default constructor for MappingRule. Used for deserialization.
+     */
+    public MappingRule() {
+    }
+
+    /**
+     * Constructor for MappingRule.
+     *
+     * @param kafkaTopicTemplate the kafka topic template.
+     * @param mqttTopicPattern   the mqtt topic pattern.
+     */
+    public MappingRule(String mqttTopicPattern, String kafkaTopicTemplate) {
+        this.kafkaTopicTemplate = kafkaTopicTemplate;
+        this.mqttTopicPattern = mqttTopicPattern;
```

Comment:
Personally would find it more intuitive if we swapped the position of `kafkaTopicTemplate` and `mqttTopicPattern` so that `mqttTopicPattern` comes first (Going from MQTT topic -> Kafka topic)

Same for the comment for this method.

Same for the comment for the class.

Reply:
fixed